### ISSUE TITLE
Not running with modified chromedriver on Mac M1

### ISSUE THREAD

**Issue by jonathanbright** [byalextran/southwest-headers#4]
I'm on a Mac with M1 (apple silicon) ARM-based processor.  There is a different chromedriver now for M1 (not the same as the amd64 driver).  This southwest-headers app runs fine until you modify the chromedriver as instructed.  I've tried doing it with the perl command given, and also manually with a hex editor.  If the driver is modified in the way described (changing the name of the cdc variable) then the program errors when an instance of the chromedriver is attempted to be created.

Here's the exact error with traceback:

Traceback (most recent call last):
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/southwest-headers.py", line 37, in <module>
    driver = webdriver.Chrome(os.getcwd() + "/chromedriver", options=chrome_options, seleniumwire_options=seleniumwire_options)
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/env/lib/python3.9/site-packages/seleniumwire/webdriver.py", line 195, in __init__
    super().__init__(*args, **kwargs)
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/env/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 70, in __init__
    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], "goog",
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/env/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 90, in __init__
    self.service.start()
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/env/lib/python3.9/site-packages/selenium/webdriver/common/service.py", line 98, in start
    self.assert_process_still_running()
  File "/Users/jonathanbright/py/Virtualenvs/southwest-headers/env/lib/python3.9/site-packages/selenium/webdriver/common/service.py", line 110, in assert_process_still_running
    raise WebDriverException(
selenium.common.exceptions.WebDriverException: Message: Service /Users/jonathanbright/py/Virtualenvs/southwest-headers/chromedriver unexpectedly exited. Status code was: -9

**Comment by byalextran** [comment]
hmm, interesting. since i don't have an M1 mac, this will be really hard to troubleshoot. it also sounds like a chromedriver issue. are you able to post this issue to the chromedriver repo? i'll try to take a look into this, but i think you'll get better luck (and hopefully a faster resolution) there.

**Comment by jonathanbright** [comment]
I'm not sure the chromedriver repo would support this with the modification being done, but maybe? 

UPDATE:  Surprisingly the mac64 version (non M1) runs fine and I was able to modify it and it ran fine as well.  I'm a bit confused by it, but it works so I went ahead and set up the cron job using the non M1 driver.  It downloads the headers fine. Whether they are valid or not, I'm still not sure how to verify.
